### PR TITLE
Getting Started translated into Russian

### LINKED ISSUE #10355: Russian Translation of Getting Started
- [x] Learn how Free Code Camp Works
- [x] Create a Github Account and Join our Chat Rooms
- [x] Configure your Code Portfolio
- [x] Join a Free Code Camp Group in Your City
- [x] Read Coding News on our Medium Publication
- [x] Learn What to Do If You Get Stuck

### PR DESCRIPTION
**Pull request by rfprod**
Closes #10355

### CODE REVIEW

**Comment by atjonathan** [comment]
please get someone else in one of the Russian chat rooms to peer review this, thanks :+1:

**Comment by rfprod** [comment]
@atjonathan hopefully someone will review this soon

**Comment by Dzheky** [comment]
I've looked at the translation and everything looks right to me.

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -817,6 +954,17 @@
               ""
             ]
           ]
+        },
+        "ru": {
+          "title": "Настройте ваше портфолио с кодом",
+          "description": [
+            [
+              "//i.imgur.com/tP2ccTE.gif",
+              "Анимация, показывающая надатие на изображения профиля в правом верхнем углу экрана от вашего портфолио с кодом и соединение с GitHub.",
```
Comment (DaniilDavletov) [inline seed/challenges/00-getting-started/getting-started.json]: опечатка: наЖатие вместо наДатие

**Comment by atjonathan** [comment]
Merging. If someone else finds a spelling mistake, they can open a new issue or pull request :+1: 
Thanks @rfprod :tada:

**Pull request merged**